ui/golemOperation/OperationDetails.tsx

```diff
@@ -0,0 +1,119 @@
+import { Flex, Text } from '@chakra-ui/react';
```

Comment:
The folder for entity operations is called `ui/entityOps`, so for consistency this one could be `ui/entityOp`.